rustls/src/msgs/handshake.rs

```diff
@@ -1635,14 +1795,8 @@ impl ServerKeyExchangePayload {
         if let Self::Unknown(ref unk) = *self {
             let mut rd = Reader::init(unk.bytes());
 
-            let params = match kxa {
```

Comment:
Nit: to minimize back and forth, I would modify the earlier commit to just inline `ServerKeyExchangeParams::Ecdh(ServerEcdhParams::read(&mut rd).ok()?)` into the `ServerKeyExchange` initialization expression.